packages/frontend/src/App.tsx

```diff
@@ -226,7 +226,10 @@ const App = () => (
                                                     >
                                                         <Projects />
                                                     </Route>
-                                                    <Route path="/home" exact>
+                                                    <Route
+                                                        path="/home/:projectUuid?"
```

Comment:
Should this be more like `/project/:projectUuid/home`? Actually not sure what best UX is for the URL bar - I'm thinking more about how I'd represent it in an API.

Reply:
eg. in sentry: 
`https://sentry.io/organizations/lightdash/projects/lightdash-server/?project=xxxxx`